test/xbgpu/test_engine.py

```diff
@@ -1482,82 +1547,68 @@ async def test_capture_stop_some_streams(
         corrprod_outputs: list[XOutput],
         beam_outputs: list[BOutput],
     ) -> None:
-        """Test capture-stop request on only some data streams.
+        """Test capture-stop request on only B-engine data streams.
 
         Issue a `?capture-stop` request at some point into data processing and
         check the corresponding streams only have partial data transmission.
-        Also ensure that data is completely received for streams that did not
-        receive a `?capture-stop` request.
+        Also ensure that data is completely received for X-engine streams that
+        did not receive a `?capture-stop` request.
+
+        This is only tested of B-engine data streams so as to deterministic in
+        measuring the strength of the test. It has been tested successfully when
+        issuing capture-stop to X-engine data streams as well.
```

Comment:
I don't follow this last paragraph at all (possibly because the first line has typos: "tested of" and "as to deterministic").

Reply:
Ah, my apologies. Quite right, I think I'd started writing a different sentence, then started inserting better wording without removing the old contents. It's meant to read more like
> This is only tested ^on B-engine data streams so as to ^be deterministic in measuring the strength of the test.

That being said, once I implement your suggested change of disabling a subset of **both** data streams then I can remove this docstring addition.